### DISCUSSION TITLE
115

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#39] [General]
# 潮流周刊第115期 - 一江两岸

封面来源于年后在老家江边散步所拍，一江清水，两岸繁华，天气不错，阳光正好。

https://weekly.tw93.fun/posts/115-%E4%B8%80%E6%B1%9F%E4%B8%A4%E5%B2%B8

**Comment by AlureLove** [comment]
请问老哥在详细介绍Alacritty的时候，可否顺便讲解下，安装的其他辅助插件等，想学习一下~

**Reply by tw93** [reply]
可见 https://tw93.fun/2023-01-25/alacritty.html

**Comment by AlureLove** [comment]
信件已收到！

**Comment by dingziqi** [comment]
大佬也是醴陵的？

**Comment by yzh990918** [comment]
真大佬
